hieradata/common.yaml

```diff
@@ -21,3 +21,11 @@ mesos_slaves:
     - crisis
 
 browser_homepage: https://www.ocf.berkeley.edu/about/lab/open-source
+
+# IPv4 and IPv6 src ranges for types of servers
+internal_zone_range_4: 169.229.226.5-169.229.226.90
+desktop_src_range_4: 169.229.226.100-169.229.226.139
+staffvm_src_range_4: 169.229.226.200-169.229.226.252
+internal_zone_range_6: 2607:f140:8801::1:5-2607:f140:8801::1:90
+desktop_src_range_6: 2607:f140:8801::1:100-2607:f140:8801::1:139
+staffvm_src_range_6: 2607:f140:8801::1:200-2607:f140:8801::1:252
```

Comment:
This bothers me a bit because the IPv6 ranges actually have gaps in them because of the unused hex digits (a-f) that we don't use to make the IPs map more easily to IPv4, but I don't know of a good way to represent that here.

Reply:
True. I don't know of a good way either...